service/lib/dinstaller/manager.rb

```diff
@@ -54,6 +58,7 @@ class Manager
     def initialize(logger)
       @logger = logger
       @status_manager = StatusManager.new(Status::Error.new) # temporary status until probing starts
+      @questions_manager = QuestionsManager.new
```

Comment:
I kind of expect that sooner or later we will need logger there to get some ideas what is going on.